app/views/helpers/injected/obligations/NextUpdatesHelper.scala.html

```diff
@@ -20,56 +20,87 @@
 @import models.nextUpdates._
 @import java.time.LocalDate
 
-@this(implicitDateFormatter: ImplicitDateFormatterImpl, govukAccordion : GovukAccordion)
+@this(implicitDateFormatter: ImplicitDateFormatterImpl, govukAccordion: GovukAccordion, govukTable: GovukTable)
 @(obligations: ObligationsModel)(implicit messages: Messages, user: auth.MtdItUser[_])
 @import implicitDateFormatter.longDate
 
-@deadlineRow(deadline: NextUpdateModelWithIncomeType) = {
-	<div class="govuk-summary-list__row">
-		<dt class="govuk-summary-list__key">
-			@{if(deadline.obligation.obligationType == "Quarterly")  messages("nextUpdates.quarterly")}
-			@{if(deadline.obligation.obligationType == "EOPS")  messages("nextUpdates.eops")}
-			@{if(deadline.obligation.obligationType == "Crystallised")  messages("nextUpdates.crystallised")}
-		</dt>
-		<dd class="govuk-summary-list__value">@{messages(deadline.incomeType)}</dd>
-	</div>
+@deadlineRow(deadline: NextUpdateModelWithIncomeType) = @{
+    Seq(
+        TableRow(
+            content = Text(getObligationTypeText(deadline.obligation.obligationType))
+        ),
+        TableRow(
+            content = Text(messages(deadline.incomeType))
+        )
+    )
 }
 
-@getHeadingMessage(obligation: NextUpdateModel) = @{
-	val key = if(obligation.obligationType == "Quarterly") "nextUpdates.section.heading.updates"
-		else "nextUpdates.section.heading.taxYear"
-	messages(key, obligation.start.toLongDate, obligation.end.toLongDate)
+@getObligationTypeText(obligation: String) = @{
+    obligation match {
+        case "Quarterly" => messages("nextUpdates.quarterly")
+        case "EOPS" => messages("nextUpdates.eops")
+        case _ => messages("nextUpdates.crystallised")
+    }
+}
+
+@getSectionHeadingMessage(obligation: NextUpdateModel) = @{
+    val key = if(obligation.obligationType == "Quarterly") "nextUpdates.section.heading.quarterly"
+    else "nextUpdates.section.heading.end-of-year"
+    messages(key)
+}
+
+@getTableHeadingMessage(obligation: NextUpdateModel) = @{
+    val key = if(obligation.obligationType == "Quarterly") "nextUpdates.section.heading.updates"
+    else "nextUpdates.section.heading.taxYear"
+    messages(key, obligation.start.toLongDateShort, obligation.end.toLongDateShort)
 }
 
 @headingContent(collectedObligations: ((LocalDate, Seq[NextUpdateModelWithIncomeType]), Int)) = {
-	<span class="govuk-!-font-size-24">
-		@{collectedObligations._1._1.toLongDate}
-	</span>
+    <span class="govuk-!-font-size-24">
```

Comment:
single parameter  is in use in the method, you can drop redundant and update caller;
also if pls avoid resolving tuple in the way as in line 61, use tuples unapply

Reply:
This wasn't my code, just reformatting. Maybe a tech debt ticket for this?

Reply:
Have you heard of the boy scout rule? Always leave code cleaner than you found it 👍 
https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html